autogpt_platform/backend/backend/copilot/tools/raise_expert.py

```
@@ -200,6 +206,28 @@ async def _execute(
                 ),
                 session_id=session_id,
             )
+        try:
+            duplicate = await _active_expert_named(user_id, params.name)
+        except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)